packages/base-driver/lib/jsonwp-proxy/proxy.js

```diff
@@ -39,7 +39,7 @@ class JWProxy {
   timeout;
 
   constructor(opts = {}) {
-    const options = _.defaults(opts, {
+    const options = _.defaults(_.omit(opts, 'log'), {
```

Comment:
this will break the existing stuff from xcuitest driver. Basically there we share the same logger instance to the jwproxy as we use in the driver, which allows to transparently connect all logs to the same driver instance.

Reply:
below we have ` this._log = opts.log;`, so it would probably make sense to have allowed properties list rather than exclude some. It would be much safer from the perspective that we assign options to `this` and we never know what `this` is going to contain

Reply:
ok @mykola-mokhnach i made a list of allowed options so we know what we're dealing with